datafusion-cli/src/highlighter.rs

```diff
@@ -0,0 +1,126 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
For some reason the RAT check fails for this file: https://github.com/apache/arrow-datafusion/actions/runs/7592786554/job/20702838429?pr=8918

I think that can typically be fixed by just copy/pasting the license text / comment from another file

Reply:
Thanks! There was a typo in the license as pointed about by @Jefffrey . Let's see if it's green now :)